client/cmd/dexc-desktop/app_darwin.go

```diff
@@ -0,0 +1,380 @@
+//go:build darwin
+
+package main
+
+/*
+dexc-desktop is the Desktop version of the DEX Client. There are a number of
+differences that make this version more suitable for less tech-savvy users.
+
+| CLI version                       | Desktop version                          |
+|-----------------------------------|------------------------------------------|
+| Installed by building from source | Installed with an installer program.     |
+| or downloading a binary. Or with  | (e.g a .dmg MacOS installer)             |
+| dcrinstall from a terminal.       |                                          |
+|-----------------------------------|------------------------------------------|
+| Started by command-line.          | Started by selecting from the start/main |
+|                                   | menu, or by selecting a desktop icon or  |
+|                                   | pinned taskbar icon. CLI is fine too.    |
+|                                   |                                          |
+|-----------------------------------|------------------------------------------|
+| Accessed by going to localhost    | Opens in WebView, a simple window        |
+| address in the browser.           | backed by a web engine.                  |
+|-----------------------------------|------------------------------------------|
+| Shutdown via ctrl-c signal.       | When user closes window, continues       |
+| Prompt user to force shutdown if  | running in the background if there are   |
+| there are active orders.          | active orders. Run a little server that  |
+|                                   | synchronizes at start-up, enabling the   |
+|                                   | window to be reopened when the user      |
+|                                   | tries to start another instance. A       |
+|                                   | desktop notification is sent and the     |
+|                                   | system tray icon remains in the tray.    |
+|-----------------------------------|------------------------------------------|
```

Comment:
macdrive looks super cool!  Let's remove this duplicate comment though.  The regular app.go can contain any design docs like this.